components/README.md

```
@@ -1,84 +1,183 @@
+### 🧱 `Modal`
 
-| Prop | Type | Default | Description |
+| Prop | Type | Default | 설명 |
 | --- | --- | --- | --- |
 | `show` | `boolean` | – | 모달 표시 여부 |
 | `onHide` | `() => void` | – | 모달 닫기 함수 |
-| `background` | `boolean` | `true` | 배경 어두움 표시 여부 |
-| `position` | `"center"` | `"top"` | `"bottom"` | `"center"` | 모달 위치 설정 |
-| `gap` | `number` | `16` | 내부 flex 간격 설정 |
+| `background` | `boolean` | `true` | 배경 클릭 시 닫기 여부 |
+| `position` | `'center' | 'top' | 'bottom'` | `'center'` | 모달 위치 |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)